src/commands/init.stanza

```diff
@@ -76,13 +77,15 @@ public defn init (cmd-args:CommandArgs) -> False:
   val main-package = to-string("%_/main" % [project-name])
   val stanza-proj-path = to-string("%_/stanza.proj" % [project-dir])
   if not file-exists?(stanza-proj-path):
-    spit(stanza-proj-path, \<>include "src/stanza.proj"
+    spit(stanza-proj-path, \<>include ".slm/stanza.proj"  ; Dependencies
+include "src/stanza.proj"  ; Project Sources
```

Comment:
at some point it might be nice to parameterize the `src` directory

Reply:
Ya - I'm not a fan of all the string literals that are bandied about in this project. I'll add a ticket in linear.